packages/notifier/test/test-publish-kit.js

```diff
@@ -650,8 +650,16 @@ const verifySubscribeAfterSequencing = test.macro(async (t, makePublishKit) => {
   const sub2LIFO = [];
 
   const sub1FirstAll = [];
-  E.when(sub1.subscribeAfter(), cell => void sub1FirstAll.push(cell), t.fail);
-  E.when(sub1.subscribeAfter(), cell => void sub1FirstAll.push(cell), t.fail);
+  void E.when(
+    sub1.subscribeAfter(),
+    cell => void sub1FirstAll.push(cell),
+    t.fail,
+  );
+  void E.when(
+    sub1.subscribeAfter(),
+    cell => void sub1FirstAll.push(cell),
+    t.fail,
+  );
```

Comment:
Consider failing the test if any part of this promise chain rejects.

```suggestion
  E.when(
    sub1.subscribeAfter(),
    cell => void sub1FirstAll.push(cell),
  ).catch(t.fail);
  E.when(
    sub1.subscribeAfter(),
    cell => void sub1FirstAll.push(cell),
  ).catch(t.fail);
```